sdk-cpp/include/do_errors.h

```diff
@@ -11,19 +11,20 @@ namespace microsoft
 {
 namespace deliveryoptimization
 {
-
-enum class errc : int32_t
+namespace errc
```

Comment:
Why change to namespace? I think one of the std::error_code constructors expects enum (maybe we aren't using that c'tor today).

Reply:
Ok I see the other changes in do_error_helpers.h now.
Ignore my comment above.

Reply:
I like it better without the typecasts